MAINTAINERS.md

```diff
@@ -0,0 +1,143 @@
+and performance optimizations.
+  * Contribute design proposals.
+
+## Code Owner
+
+A Code Owner has responsibility for a specific project component or a functional
+area. Code Owners are collectively responsible, with other Code Owners,
+for developing and maintaining their component or functional areas, including
+reviewing all changes to corresponding areas of responsibility and indicating
+whether those changes are ready to be merged. Code Owners have a track record of
+contribution and review in the project.
+
+Requirements:
+  * Track record of accepted code contributions to a specific project component.
+  * Track record of contributions to the code review process.
+  * Demonstrated in-depth knowledge of the architecture of a specific project
+    component.
+  * Commits to being responsible for that specific area.
+
+The process of becoming a Code Owner is:
+1. A Contributor is nominated by opening a PR modifying the MAINTAINERS.md file
+including name, Github username, and affiliation.
+2. At least two specific component Maintainers approve the PR.
+3. CODEOWNERS.md file is updated to represent corresponding areas of responsibility.
+
+## Maintainer
+Maintainers are the most established contributors who are responsible for the 
+project technical direction and participate in making decisions about the
+strategy and priorities of the project.
+
+Requirements:
+  * Experience as a Code Owner.
+  * Track record of major project contributions to a specific project component.
+  * Demonstrated deep knowledge of a specific project component.
+  * Demonstrated broad knowledge of the project across multiple areas.
+  * Commits to using priviledges responsibly for the good of the project.
+  * Is able to exercise judgment for the good of the project, independent of
+    their employer, friends, or team.
+
+Process of becoming a maintainer:
+1. A Maintainer may nominate a current code owner to become a new Maintainer by 
+opening a PR against MAINTAINERS.md file.
+2. A majority of the current Maintainers must then approve the PR.
+
+# Code Owners and Maintainers List
+
+## oneTBB core (API, Architecture, Tests)
+
+| Name                  | Github ID             | Affiliation       | Role       |
+| --------------------- | --------------------- | ----------------- | ---------- |
+| Konstantin Boyarinov  | @kboyarinov           | Intel Corporation | Code Owner |
+| Aleksei Fedotov       | @aleksei-fedotov      | Intel Corporation | Code Owner |
+| Ilya Isaev            | @isaevil              | Intel Corporation | Code Owner |
+| Sarath Nandu R        | @sarathnandu          | Intel Corporation | Code Owner |
+| Dmitri Mokhov         | @dnmokhov             | Intel Corporation | Code Owner |
+| Michael Voss          | @vossmjp              | Intel Corporation | Maintainer |
+| Alexey Kukanov        | @akukanov             | Intel Corporation | Code Owner |
+| Pavel Kumbrasev       | @pavelkumbrasev       | Intel Corporation | Code Owner |
```

Comment:
Maybe sort this list by Role?

Reply:
I suggest sorting the list by name as those are usually unique and does not depend on the account ID in case the project would move to another platform someday.

Also, what about e-mail addresses of people in the list? Do we want to list them here as well so that anyone interested can reach out to them in case of a need or we think that tagging a person in a GitHub Issue is enough?